src/main/webapp/CreateProfilePage.html

```diff
@@ -1,17 +1,18 @@
 <!DOCTYPE html>
 <html>
-  <body>
-    <h1>Create Your Profile Page</h1>
-    <form action="/profile-data" method="POST">
-      <label for="name">Name:</label><br />
-      <input type="text" name="name" maxlength="25"/><br />
-      <label for="email">Email:</label><br />
-      <input type="email" name="email" maxlength="25"/><br />
-      <label for="username">Username:</label><br />
-      <input type="text" name="username" maxlength="20"/><br />
-      <label for="bio">Bio:</label><br />
-      <input type="text" name="bio" maxlength="160"/><br />
-      <input type="submit" value="Submit" />
+
+<body>
+	<h1>Create Your Profile Page</h1>
+	<form action="/profile-data" method="POST">
+		<label for="name">Name:</label><br />
+		<input type="text" name="name" maxlength="25"/><br />
+		<label for="email">Email:</label><br />
+		<input type="email" name="email" maxlength="25"/><br />
+		<label for="username">Username:</label><br />
+		<input type="text" name="username" maxlength="20"/><br />
```

Comment:
Make sure that indentation has 2 spaces instead of 4. I would run prettier to format the html files.